src/dispatch/individual/service.py

```diff
@@ -105,36 +93,18 @@ def update(
 ) -> IndividualContact:
     individual_contact_data = jsonable_encoder(individual_contact_in)
 
-    project = project_service.get_by_name(
-        db_session=db_session, name=individual_contact_in.project.name
-    )
-
-    terms = [
-        term_service.get_or_create(db_session=db_session, term_in=t)
-        for t in individual_contact_in.terms
-    ]
-    incident_priorities = [
-        incident_priority_service.get_by_name(
-            db_session=db_session, project_id=project.id, name=n.name
-        )
-        for n in individual_contact_in.incident_priorities
+    filters = [
+        search_filter_service.get(db_session=db_session, search_filter_id=f.id)
+        for f in individual_contact_in.filters
     ]
-    incident_types = [
-        incident_type_service.get_by_name(db_session=db_session, project_id=project.id, name=n.name)
-        for n in individual_contact_in.incident_types
-    ]
-    update_data = individual_contact_in.dict(
-        skip_defaults=True, exclude={"terms", "incident_priorities", "incident_types", "project"}
-    )
+
+    update_data = individual_contact_in.dict(skip_defaults=True, exclude={"filters"})
 
     for field in individual_contact_data:
         if field in update_data:
             setattr(individual_contact, field, update_data[field])
 
-    individual_contact.terms = terms
-    individual_contact.incident_types = incident_types
-    individual_contact.incident_priorities = incident_priorities
-    individual_contact.project = project
```

Comment:
Are individuals not scoped to projects? I can't remember whether we said we would scope them to projects or not.

Reply:
They are, they are just added on creation here: https://github.com/Netflix/dispatch/blob/feature/user-engagement-filters/src/dispatch/individual/service.py#L79